### DISCUSSION TITLE
[FR] Add subreddit social on Reddit

### DISCUSSION THREAD

**Discussion by NurMiApp** [hushenghao/AndroidEasterEggs#363] [Ideas]
### Checklist / 检查清单

- [X] No one has submitted a similar or identical feature request before. / 之前没有人提交过类似或相同的功能请求。
- [X] This suggestion does not depart from the original intention of Easter Eggs. / 这个建议不会背离 Easter Eggs 的初衷。

### Enhancement propose / 改进目的

At the Settings icons does it was this "Reddit" a available.

### Additional info / 额外信息

Huawei BAH2-W12 / Android 8.0

**Comment by hushenghao** [comment]
I'm not sure if I want to do this, I think it's better to put it in the "discussion"

**Reply by NurMiApp** [reply]
At r/reclassified on this r/AndroidEasterEggs banned. Yeah?
